test/hotspot/jtreg/compiler/c2/aarch64/TestSVEWithJNI.java

```diff
@@ -26,7 +26,7 @@
 /**
  * @test
  *
- * @requires os.arch == "aarch64" & vm.compiler2.enabled
+ * @requires os.arch == "aarch64" & os.family == "linux" & vm.compiler2.enabled
```

Comment:
I have filled separate bug for this https://bugs.openjdk.java.net/browse/JDK-8254670 as it's not supposed to be a  part of jep388/391

Reply:
Let's keep it here until the fix lands in master and we can merge it back into this branch.